cmd/kube-proxy/app/server.go

```diff
@@ -344,6 +349,12 @@ with the apiserver API to configure the proxy.`,
 			verflag.PrintAndExitIfRequested()
 			utilflag.PrintFlags(cmd.Flags())
 
+			// This can happen only on Windows if it cannot register or unregister
+			// as a Windows service. On Linux err will always be nil.
+			if err := initForOS(); err != nil {
+				glog.Fatalf("unable to register or unregister as a service: %v", err)
```

Comment:
If dropping the register/unregister flags, maybe this error message should change as well?